keyboards/ymdk/ymd09/keymaps/vial/rules.mk

```diff
@@ -5,3 +5,5 @@ QMK_SETTINGS = no
 TAP_DANCE_ENABLE = no
 COMBO_ENABLE = no
 KEY_OVERRIDE_ENABLE = no
+VIALRGB_ENABLE = yes
+RGB_MATRIX_ENABLE = yes
```

Comment:
```suggestion
```
Isn't needed since it's already enabled in `ymd09/info.json`.

Reply:
You are of course right, I've incorporated your comment into the PR.